net/scripts/colo-node-onfree.sh

```diff
@@ -4,15 +4,15 @@
 SOLANA_LOCK_FILE="${SOLANA_LOCK_FILE:?}"
 SECONDARY_DISK_MOUNT_POINT="${SECONDARY_DISK_MOUNT_POINT:?}"
 SSH_AUTHORIZED_KEYS="${SSH_AUTHORIZED_KEYS:?}"
-FORCE_DELETE="${FORCE_DELETE}"
+NEED_TO_FORCE_DELETE="${FORCE_DELETE}"
```

Comment:
is this intended to be an unset/empty check instead?

Reply:
```suggestion
FORCE_DELETE="${FORCE_DELETE:?}"
```

Reply:
yeah. Originally I guessed we didn't use `${FORCE_DELETE?}"` cuz maybe it's unset for some reason.
I just searched our code again and found it only used at https://github.com/solana-labs/solana/blob/master/net/scripts/colo-utils.sh#L242-L249
Maybe it is harmless to use `${FORCE_DELETE?}`

Reply:
fixed [3935ed1](https://github.com/solana-labs/solana/pull/27463/commits/3935ed105058b793390a514173782021de561ee9)